tests/integ/test_column.py

```diff
@@ -216,3 +216,86 @@ def test_column_with_builtins_that_shadow_functions(session):
     with pytest.raises(TypeError) as ex_info:
         TestData.double1(session).select(sum(col("a"))).collect()
     assert iter_error_msg_text in str(ex_info)
+
+
+@pytest.mark.parametrize("enable_sql_simplifier", [True, False])
```

Comment:
We already have two merge-gates where sql simplifier is enabled and disabled. I think this is redundant